_format/1.1/index.md

```diff
@@ -190,8 +192,18 @@ Here's how an article (i.e. a resource of type "articles") might appear in a doc
 
 #### <a href="#document-resource-object-identification" id="document-resource-object-identification" class="headerlink"></a> Identification
 
-Every [resource object][resource objects] **MUST** contain an `id` member and a `type` member.
-The values of the `id` and `type` members **MUST** be strings.
+As noted above, every [resource object][resource objects] **MUST** contain a
+`type` member. Every resource object **MUST** also contain an `id` member,
+except when the resource object originates at the client and represents a new
+resource to be created on the server. The values of the `id` and `type` members
+**MUST** be strings.
+
+A [resource object][resource objects] **MAY** contain a `lid` member that,
+together with `type`, uniquely identifies the resource _locally_ within the
+document. If a [resource object][resource objects] contains a `lid` member, then
+every representation of that resource, including other [resource objects] and
```

Comment:
missing word: "...of that resource, including **within** other..."

Reply:
I don't think that word is missing? Because:
> including other ROs and RIOs must also *contain* […]

("including within other ROs and RIOs must also contain" does not sound right to me)

Reply:
The full sentence:

> If a resource object contains a lid member, then every representation of that resource, including other resource objects and [resource identifier objects], MUST also contain the lid member with the same value.

I am suggesting clarifying that the other resources objects are _referring_ to the subject of the sentence, not that they themselves are other versions of it (which is prohibited by JSON API).

Here's another go, even more specific:

> If a resource object contains a lid member, then every representation of that resource, including __referential representations contained within__ other resource objects and [resource identifier objects], MUST also contain the lid member with the same value.